### ISSUE TITLE
Integrated test runner does not show test message

### ISSUE THREAD

**Issue by stefan-toubia** [BetterThanTomorrow/calva#425]
`clojure.test/is` allows you to include a message in the test that would display in the test output if the test fails.  This would be helpful to include in the Calva says output as well.

Example:
```
(testing "Test message"
      (is (= 1 2) "Math is hard"))
```
Calva says:
```
FAIL in: upload_helpers_test.clj: 21: create-s3-key-test: Test message:
  expected: 1

  actual: (2)
```
CircleCI:
```
FAIL in upload-helpers-test/ (create-s3-key-test) (upload_helpers_test.clj:21)
Test message
Math is hard
expected: 1
  actual: (2)
```

**Comment by PEZ** [comment]
Thanks! I think this should be fairly easy to fix. Please feel invited to have a look at it, if you have the time.

**Comment by adairdavid** [comment]
Hi @PEZ! It looks like this issue has been resolved as of #470 (merged Nov 14, 2019).

Was just looking around at good first issues, but looks like this is done already!

Thanks for all your work on Calva! It's a great addition to the Clojure ecosystem.
